x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/test_now_mode/test_now_mode_flyout.tsx

```diff
@@ -97,7 +97,7 @@ export function TestNowModeFlyout({
     </EuiFlyout>
   );
 
-  return <>{(testRun || inProgress) && <EuiErrorBoundary>{flyout}</EuiErrorBoundary>}</>;
+  return <>{(testRun || inProgress) && <KibanaErrorBoundary>{flyout}</KibanaErrorBoundary>}</>;
```

Comment:
Same as above

Reply:
Oops, I have a commit that already removes _this_ boundary, sorry, I will push soon. I must have forgotten to push the latest changes when I made the PR 👎

Reply:
I think in this case we can remove the boundary altogether and rely on the page-level boundary. We will have a section boundary nested inside the flyout, if something happens outside of that scope then we can consider it a page-level error and bubble up to the higher one you mention in the above comment.